client/me/purchases/manage-purchase/auto-renew-toggle/index.jsx

```diff
@@ -139,16 +143,31 @@ class AutoRenewToggle extends Component {
 		return this.props.isEnabled;
 	}
 
+	renderTextStatus() {
+		const { translate, isEnabled, withPlan } = this.props;
+
+		if ( withPlan ) {
+			return isEnabled ? translate( 'Plan auto-renew (on)' ) : translate( 'Plan auto-renew (off)' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Does my plan auto-renew?' )` *ES Score: 7*